specs/vep-1739-multiple-python-versions/diagrams/high_level_sequence_plantUML_source.txt

```diff
@@ -0,0 +1,18 @@
+@startuml
+actor DataEngineer as engineer
+participant ControlService as cs
+participant K8sNamespace_DataJobBuilders as namespace
+database Database as db
+database Registry as ecr
+
+engineer -> cs : vdk create
+cs -> db : Register new data job
+cs -> engineer : Download sample data job
+engineer -> engineer : Set <b>python_version</b> in the data job's config.ini
+engineer -> cs : vdk deploy
+cs -> cs : Read Deployment data
+cs -> db : Update data job configuration data
+cs -> ecr : Pull data job base image
```

Comment:
Consider using activation to show a group of operation being done together. 

Consider using return arrow when the response is returned and outline use an arrow to indicate when async operation starts (different type of arrows are explained here - https://www.lucidchart.com/pages/uml-sequence-diagram .

Reply:
Modified the diagram based on the suggestions.